dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganisationResource.java

```diff
@@ -362,6 +362,19 @@ public Organisation updateOrganisation(@ApiParam(hidden = true) @Auth User user,
         return organisationDAO.findById(id);
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @Path("/{organisationId}/users/{username}")
+    @ApiOperation(value = "Adds a user role to an organisation.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = OrganisationUser.class)
+    public OrganisationUser addUserToOrgByUsername(@ApiParam(hidden = true) @Auth User user,
```

Comment:
What is the reason for adding by username and not ID? Is the username unique?

Reply:
By ID is impossible to use.  Username is unique.